repo/linux/andi-github

```diff
@@ -0,0 +1,6 @@
+url: https://github.com/ashyti/linux
```

Comment:
thanks, is it ok to name this repo as ashyti? We have one rule [1] for repo name, though it is not that obvious.

[1] https://github.com/intel/lkp-tests/wiki/Repo-Naming-Convention

Reply:
I'm OK with ashyti... thanks!

Do you want me to send a new pull request?

Thanks a lot!
Andi